etc/profile-m-z/pycharm-community.profile

```diff
@@ -30,7 +36,6 @@ tracelog
 
 # private-etc alternatives,fonts,passwd - minimal required to run but will probably break
 # program!
-private-cache
 private-dev
```

Comment:
Just for the record - why did `private-cache` need removing?

Reply:
Because PyCharm needs persistent cache for spell and grammar checkers, etc (i. e. if you download spelling dictionary with `private-cache`, on restart PyCharm you need to download spelling dictionary again).

Reply:
@k4leg [on Nov 23](https://github.com/netblue30/firejail/pull/5476#discussion_r1030686066):

> Because PyCharm needs persistent cache for spell and grammar checkers, etc
> (i. e. if you download spelling dictionary with `private-cache`, on restart
> PyCharm you need to download spelling dictionary again).

I see, that makes sense.

Just make sure to put these details in the commit message, so that the reason
for the change is clear.  If is unclear, eventually somebody might assume that
it was done by accident and undo it, for example.

See the following for details:

* <https://chris.beams.io/posts/git-commit/>